src/parallel_matmul.jl

```diff
@@ -3,14 +3,6 @@
 # implement A_mul_B* with normal vectors, not just shared arrays, for sharedsparsematrix
 # implement load balancing for multiplication
 
-import
-    Base.A_mul_B, Base.At_mul_B, Base.Ac_mul_B, Base.A_mul_B!, Base.At_mul_B!, Base.Ac_mul_B!, 
-    Base.sdata, Base.size, Base.display
-
-export
-    SharedBilinearOperator, SharedSparseMatrixCSC, 
-    share, display, sdata, operator, nfilled, size,
-    A_mul_B, At_mul_B, Ac_mul_B, A_mul_B!, At_mul_B!, Ac_mul_B!, *
```

Comment:
If you're importing a function from Base you technically don't need to reexport that function.